pkg/uninstaller/resources.go

```diff
@@ -0,0 +1,200 @@
+package uninstaller
+
+import (
+	"context"
+	clusterconfigV1alpha1 "github.com/liqotech/liqo/apis/config/v1alpha1"
+	discoveryV1alpha1 "github.com/liqotech/liqo/apis/discovery/v1alpha1"
+	"github.com/liqotech/liqo/apis/net/v1alpha1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/klog/v2"
+	"sync"
+	"time"
+)
+
+type toCheckDeleted struct {
+	gvr           schema.GroupVersionResource
+	labelSelector metav1.LabelSelector
+}
+
+type resultType struct {
+	Resource toCheckDeleted
+	Success  bool
+}
+
+var (
+	podGVR = schema.GroupVersionResource{
+		Group:    "",
+		Version:  "v1",
+		Resource: "pods",
+	}
+
+	toCheck = []toCheckDeleted{
+		{
+			gvr:           v1alpha1.TunnelEndpointGroupVersionResource,
+			labelSelector: metav1.LabelSelector{},
+		},
+		{
+			gvr:           v1alpha1.NetworkConfigGroupVersionResource,
+			labelSelector: metav1.LabelSelector{},
+		},
+		{
+			gvr: podGVR,
+			labelSelector: metav1.LabelSelector{
+				MatchLabels: map[string]string{"app": "virtual-kubelet"},
```

Comment:
Those labels should probably not be hardcoded, otherwise issues will arise as soon as they are changed